netbox/extras/lookups.py

```diff
@@ -14,4 +15,18 @@ def as_sql(self, qn, connection):
         return 'CAST(LENGTH(%s) AS BOOLEAN) != %s' % (lhs, rhs), params
 
 
+class NetContainsOrEquals(Lookup):
+    """
+    This lookup has the same functionality as the one from the ipam app except lhs is cast to inet
+    """
+    lookup_name = 'net_contains_or_equals'
+
+    def as_sql(self, qn, connection):
+        lhs, lhs_params = self.process_lhs(qn, connection)
+        rhs, rhs_params = self.process_rhs(qn, connection)
+        params = lhs_params + rhs_params
+        return 'CAST(%s as inet) >>= %s' % (lhs, rhs), params
```

Comment:
Just for readability
```suggestion
        return 'CAST(%s AS INET) >>= %s' % (lhs, rhs), params
```